libs/dtm/dtm_recovery.py

```diff
@@ -134,28 +141,115 @@ def perform_ops(self, workload_info: list, queue, skipread: bool = True,
             queue.put([False, f"S3bench workload for failed."
                               f" Please read log file {log_path}"])
 
-    def process_restart(self, master_node, pod_prefix, container_prefix, process,
-                        recover_time: int = 30):
+    # pylint: disable-msg=too-many-locals
+    def process_restart(self, master_node, ha_obj, pod_prefix, container_prefix, process,
+                        process_ids: list = None, check_proc_state: bool = False,
+                        proc_state: str = expected_process_state):
         """
         Restart specified Process of specific pod and container
         :param master_node: Master node object
+        :param ha_obj: Master node ha object
```

Comment:
is ha_obj for master node or this is for HA lib as the poll_cluster_status is in lib and if that's the case, why do we want it as param, we can import lib directly isn't it?

Reply:
yes, we can directly import also.